batavia/types/StrUtils.js

```diff
@@ -1233,11 +1230,15 @@ function _new_subsitute(str, args, kwargs) {
         // error for converting floats with improper presentation types
         // TODO: need to check for decimal once it is implemented
         if (types.isinstance(this.arg, [types.Float]) && /[bcdoxX]/.test(type)) {
-            throw new types.ValueError.$pyclass(`Unknown format code '${type}' for object of type '${type_name(this.arg)}'`)
+            throw new exceptions.ValueError.$pyclass(`Unknown format code '${type}' for object of type 'float'`)
```

Comment:
The .eslintrc disallows template strings. You didn't introduce this, and because the eslint submit checks aren't working this won't be flagged, but if you could convert it to a regular string with concatenation, that'd cool.